src/main/docker/Dockerfile.jdk

```diff
@@ -26,7 +26,8 @@ FROM ${DOCKER_HUB_PRIVATE}/cafinternal/prereleases:opensuse-base-2.2.0-SNAPSHOT
 # Refresh the OS repositories and install OpenJDK 8 Development Kit
 RUN zypper -n refresh && \
     zypper -n update && \
-    zypper -n install java-1_8_0-openjdk-devel
+    zypper -n install java-1_8_0-openjdk-devel && \
+    zypper al java-1_8_0-openjdk
```

Comment:
Should this be `java-1_8_0-openjdk-devel`?

Reply:
`java-1_8_0-openjdk` covers the devel one too as well as other that start with `java-1_8_0-openjdk`, for example:
```
550fa87369cf:/ # zypper update
Loading repository data...
Reading installed packages...

The following item is locked and will not be changed by any action:
 Installed:
  java-1_8_0-openjdk

The following 3 package updates will NOT be installed:
  java-1_8_0-openjdk java-1_8_0-openjdk-devel java-1_8_0-openjdk-headless

The following NEW package is going to be installed:
  libyaml-cpp0_6

The following 54 packages are going to be upgraded:
  aaa_base ca-certificates-mozilla curl diffutils file file-magic findutils glibc krb5 libasound2 libblkid1 libcom_err2 libcurl4 libfdisk1 libfreebl3 libfreetype6 libgcc_s1 libgnutls30 libjpeg8 libldap-2_4-2 libldap-data libmagic1
  libmount1 libopenssl1_1 libpq5 libprocps7 libproxy1 libsmartcols1 libsoftokn3 libsolv-tools libstdc++6 libsystemd0 libudev1 libuuid1 libX11-6 libX11-data libxcb1 libxml2-2 libz1 libzypp mozilla-nspr mozilla-nss mozilla-nss-certs
  openssl-1_1 openSUSE-release openSUSE-release-appliance-docker pam permissions postgresql postgresql10 procps sysuser-shadow util-linux zypper

The following product is going to be reinstalled:
  "openSUSE Leap 15.1"

The following 2 packages require a system reboot:
  glibc libopenssl1_1

54 packages to upgrade, 1 new.
Overall download size: 22.1 MiB. Already cached: 0 B. After the operation, additional 201.3 KiB will be used.

    Note: System reboot required.
```

Reply:
Having said that I just tested `java-1_8_0-openjdk-devel` and it works the same way so either can be used.